capstone/src/main/java/com/google/univiz/UniversityData.java

```diff
@@ -0,0 +1,110 @@
+package com.google.univiz;
+
+public final class UniversityData {
+
+  public final class UniversityBuilder {
+    private String uniName;
+    private String uniCity;
+    private float latitude;
+    private float longitude;
+    private int isMainCampus;
+    private int urbanizationDegree;
+    private float admissionRate;
+    private float avgSAT;
+    private int numOfUndergrads;
+    private int avgCostOfAttendance;
+    private int numOfStudentsInDebt;
+    private int carnegieSizeDegree;
+    private float numOfUndergradMen;
+    private float numOfUndergradWomen;
```

Comment:
Instead of setting the properties of the builder, you can do this instead:
```suggestion
public final class UniversityData {

  private String uniName;
  private String uniCity;
  private float latitude;
  private float longitude;
  private int isMainCampus;
  private int urbanizationDegree;
  private float admissionRate;
  private float avgSAT;
  private int numOfUndergrads;
  private int avgCostOfAttendance;
  private int numOfStudentsInDebt;
  private int carnegieSizeDegree;
  private float numOfUndergradMen;
  private float numOfUndergradWomen;
  public final class UniversityBuilder {
    UniversityData uniData = new UniversityData();
```
and set the properties of `uniData` instead. Then in the build function, you can return `uniData`.

Reply:
do i keep the ```uniData.uniName = this.uniName``` lines? or do i just do ```return uniData```?

Reply:
[This](https://www.vogella.com/tutorials/DesignPatternBuilder/article.html) seems like a good ref. Not sure if this is what Ihsan is getting at, but it's one way of doing it.

Reply:
Just `return uniData`.

Reply:
ok i will try that.

Reply:
> still getting these `cannot find symbol` errors, just now in the `UniversityData` class
> ![image](https://user-images.githubusercontent.com/51244558/88987260-3a150f80-d28a-11ea-9c5d-2e7455e9f25b.png)

I forgot to mention, every instance of `this.property` would now be replaced by `uniData.property.`